elrond-wasm-node/src/api/send_api_node.rs

```diff
@@ -0,0 +1,211 @@
+use super::ArwenBigUint;
+use crate::ArwenApiImpl;
+use elrond_wasm::api::SendApi;
+use elrond_wasm::types::{Address, ArgBuffer, BoxedBytes, CodeMetadata};
+
+extern "C" {
+	fn transferValue(
+		dstOffset: *const u8,
```

Comment:
Optionally add `type ExternalPointer = *const u8;` somewhere accessible to all API components, and use it everywhere, if it makes code clearer.